Jenkinsfile.tests

```diff
@@ -0,0 +1,88 @@
+pipeline {
+    agent {
+        label 'buildagent'
+    }
+    parameters {
+        string(name: 'ACTION', defaultValue: 'it', description: '')
+    }
+    environment {
+        SBT_HOME = tool name: 'sbt-1.2.6', type: 'org.jvnet.hudson.plugins.SbtPluginBuilder$SbtInstallation'
+        SBT_THREAD_NUMBER = 6
```

Comment:
Can we put threads number in env? It might be useful to increase this number without dedicated pull request

Reply:
done